rome_app/models.py

```diff
@@ -72,6 +72,29 @@ class Essay(models.Model):
     pids = models.CharField(max_length=254, null=True, blank=True, help_text='Comma-separated list of pids for books or prints associated with this essay.')
     people = models.ManyToManyField(Biography, null=True, blank=True, help_text='List of people associated with this essay.')
 
+    def preview(self):
+        return self.text[:254]
+
+    def related_works(self):
+        num_prints_estimate = 6000
+        if self.pids is None:
```

Comment:
I'd suggest using "if not self.pids:" here. That will handle None and an empty string.